README.md

```diff
@@ -14,6 +14,7 @@ There are already several R packages based on **htmlwidgets**, including:
 * [sparkline](https://github.com/htmlwidgets/sparkline) --- Small inline charts
 * [DT](http://rstudio.github.io/DT/) --- Tabular data via DataTables
 * [rthreejs](https://github.com/bwlewis/rthreejs) -- Interactive 3D graphics
+* [cofeewheel](https://github.com/bwlewis/rthreejs) -- Interactive hierarchical data visualization with D3 Sunburst
```

Comment:
argh! Apparently I forgot to change the URL after I copied and pasted. My bad, will be fixed in the next request.

Thanks,
-- Arman